lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -8275,9 +8275,10 @@ bool SwiftASTContext::CacheUserImports(SwiftASTContext &swift_ast_context,
   llvm::SmallString<1> m_description;
   llvm::SmallVector<swift::ModuleDecl::ImportedModule, 2> parsed_imports;
 
-  swift::ModuleDecl::ImportFilter import_filter;
-  import_filter |= swift::ModuleDecl::ImportFilterKind::Public;
-  import_filter |= swift::ModuleDecl::ImportFilterKind::Private;
+  swift::ModuleDecl::ImportFilter import_filter = {
+    swift::ModuleDecl::ImportFilterKind::Public,
+    swift::ModuleDecl::ImportFilterKind::Private
+  }
```

Comment:
Missing semicolon?

Reply:
@varungandhi-apple Fixed! Sorry about that.